app/routes/index.tsx

```diff
@@ -54,19 +64,37 @@ export default function Index(): JSX.Element {
 	return (
 		<>
 			<Navbar groupedPages={groupedPages} />
-			<Container w="full" maxW="7xl" mx="auto" py={8} centerContent>
-				<chakra.section>
+			<Container
+				w="full"
+				// bg="#f6f9fc"
+				bg={useColorModeValue("gray.50", "")}
+				maxW="full"
+				mx="auto"
+				py={8}
+				pos="relative"
+				centerContent>
+				<chakra.section pb={8} pt={4}>
 					<Hero />
 				</chakra.section>
+				<Awards />
+			</Container>
+			<Container w="full" maxW="full" mx="auto" py={8} centerContent>
 				<chakra.section py={16}>
-					<Box textAlign="center">
+					<Box textAlign="center" mb="8">
 						<Heading as="h1" size="2xl">
-							Home
+							Aktuelle Termine
 						</Heading>
-						<Text fontSize="lg">
-							Hier geht&apos;s irgendwann weiter!
+						<Text fontSize="xl" mt="3">
+							Alle demnächst anstehende Termine des
+							Louise-Schroeder-Gymnasiums:
 						</Text>
 					</Box>
+					<Calendar />
+					<Box textAlign="center" mt="8">
+						<Link href="/" color="rgb(0, 119, 255)">
```

Comment:
Overwriting the color of a link is not good for accessibility and from a UX standpoint, this should be left up to chakra and the browser. If we truly need a different link color, this should be changed altogether in the theme config.